FNF_MissionTemplate.VR/server/init/fn_adminPatch.sqf

```diff
@@ -0,0 +1,48 @@
+/*
+ * Author: tanaKa and Indigo
+ * Adds patches on admins for FNF
+ *
+ * Params: 
+ * N/A
+ *
+ * Return value:
+ * N/A
+ *
+ * Usage: 
+ * call PHX_fnc_adminPatch
+ *
+ * Public: no
+ */
+
+_ids = [
+  "76561198011678508", // muniz
+  "76561197961336958", // arkor
+  "76561197991996737", // indigo
+  "76561198142848471", // dingo
+  "76561198046071194", // wheaton
+  "76561198046962658", // mart
+  "76561197989932191", // anderson
+  "76561198058373929", // fire
+  "76561198136175919", // strac
+  "76561198068436818", // bridges
+  "76561197960285643", // nemesis
+  "76561197977759652", // drake
+  "76561198097983515", // dream
+  "76561198061743924", // mallen
+  "76561197967346068", // cullen
+  "76561198144278271", // pads/pickles
+  "76561198060054975", // falenty
+  "76561198118254864", // fred
+  "76561198839019295", // dusty
+  "76561198054866659", // rory
+  "76561197986258732"  // tanaKa
+];
+
+if (isServer) then {
+	{
+		if ((getPlayerUID _x) in _ids) then {
+			[_x, "FNFPatchAdmin"] call bis_fnc_setUnitInsignia;
+			"[FNF Patches] Assigned admin patch" remoteExec ["systemChat", _x];
```

Comment:
I think this remoteExec would systemChat for everyone, N times. Recommend moving patch setup to be client side and do a local systemChat if player is admin. Or, at least remoteExec each message to the right player.

Reply:
> The server runs an addon that sends the clients an array of staff members https://github.com/FridayNightFight/FNF/blob/dev/addons_server/fnf_server_vars/staffList.sqf
> Var is `fnf_staffInfo`, so use that. You'll then be able to run `PHX_fnc_adminPatch` on the clients instead.

Changing to var now. Why make it run on clients? ``bis_fnc_setUnitInsignia`` is global and is recommended to only be called on servers.

Reply:
> I think this remoteExec would systemChat for everyone, N times. Recommend moving patch setup to be client side and do a local systemChat if player is admin. Or, at least remoteExec each message to the right player.

The target of the remoteExec is only the player of the UID in the loop, but it could definitely use a locality check with multiple players on the server.

Reply:
Ah, my bad, I misread the remoteExec params. This looks good as is.

Running it client side would make the code a little simpler, and the doc examples are client side, but it should work either way.